ESP/tools/createzip.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+Import("env")
+
+import sys
+from ntpath import basename
+from zipfile import ZipFile
+import json
+
+
+def createZip(source, target, env):
+    if (
+        sys.platform.startswith("ubuntu")
+        or sys.platform.startswith("Linux")
+        or sys.platform.startswith("linux")
+    ):
+        print("Program has been built, creating zip archive!")
+        program_path = target[0].get_abspath()
+        #print("binary path", program_path)
+        #print("FLASH_EXTRA_IMAGES: %s\n" % env["FLASH_EXTRA_IMAGES"])
+        #print("ESP32_APP_OFFSET: %s\n" % env["ESP32_APP_OFFSET"])
+        
+        array_args = [env["FLASH_EXTRA_IMAGES"]]
+
+        for [offset, image] in env["FLASH_EXTRA_IMAGES"]:
```

Comment:
`[]` is redundant here, we can safely remove it, python will do the unpacking either way, for example:

```python
t = [(1,2)]
for num1, num2 in t:
    print(num1, num2)  
```

Reply:
Alright :) 👍